code/src/components/Summary.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import arrow from './listarrow.png'
+
+// I wanted to make a bullet list with images instead of dots using list-style-image.
+// But list-style-image doesn't work with display:flex and I couldn't
+// align the text right without it so I ended up putting the image in the list.
+
+export const Summary = ({ name, home, lenght, bike, want }) => {
+  return (
+    <ul className="summary">
+      <li><img src={arrow} alt="arrow icon" />Your name is {name}.</li>
```

Comment:
Maybe one way to get the code slightly shorter would be to do a const for the image? Or maybe some kind of map-function? Not sure how though, sorry!